### PR TITLE
feat(splitfullname): implement utility function to separate fullname

### PR DESCRIPTION
**Pull request by svandu**
feature #29 

@subrat611

### CODE REVIEW

**Comment by subrat611** [comment]
After reviewing and passing through all test cases. I will merge this PR.

**Comment by subrat611** [comment]
@svandu

Please update your branch with the main branch.

Then run npm run test ./utils/__tests__ in your terminal.

File: utils/splitFullName.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+function splitFullName(fullName) {
+  let firstName, lastName;
+  let fullNameArray = fullName.split(" ");
+
+  if (fullName && fullNameArray.length > 1) {
+    const [firstNameArray, ...lastNameArray] = fullNameArray;
+    firstName = firstNameArray;
+    lastName = lastNameArray.join("");
+  } else {
+    if (fullName == null) {
```
Comment (subrat611) [inline utils/splitFullName.js:10]: Why are you checking `fullName` here again?

It has already been checked at top-level condition.

**Comment by svandu** [comment]
All test cases pass

image

@subrat611

**Review by subrat611 — APPROVED** [review]

**Pull request merged**